learn/getting-started/getting-started-with-ballerina.md

```diff
@@ -1,74 +1,84 @@
 
-To complete this tutorial, you need:
+To complete this tutorial, you need the below.
 
-- A command terminal.
-- A text editor. VSCode](https://code.visualstudio.com/) with the [Ballerina extension](https://marketplace.visualstudio.com/items?itemName=WSO2.ballerina) installed is our preferred choice, and it has good support for Ballerina.
-- A Ballerina installation. Follow the steps in [Installing Ballerina](https://ballerina.io/learn/installing-ballerina/setting-up-ballerina/).
-- Some experience with at least one programming language. 
+1. A command terminal
+2. A text editor
+    >**Tip:** Prefarably, [Visual Studio Code](https://code.visualstudio.com/) with the [Ballerina extension](https://marketplace.visualstudio.com/items?itemName=WSO2.ballerina) installed as it has good support for Ballerina.
+3. A [Ballerina installation](https://ballerina.io/learn/installing-ballerina/setting-up-ballerina/)
+4. Prior programming experience
 
-## Meet `bal`
+## Meeting `bal`
 
-`bal` is the Ballerina build tool and package manager. Among other things, `bal` helps you to create, build, test, and run your project. 
+`bal` is the Ballerina build tool and package manager. Among other things, `bal` helps you to create, build, test, and run your project. The latest `bal` tool version is available with the latest Ballerina installation. 
 
-You can get the latest `bal` tool version with the Ballerina installation. Open your terminal and run the following commands just to make sure everything is ready:
+Open your terminal, and run the following commands to make sure everything is ready.
 
 ```bash
 $bal
 $bal version
 ```
 
-## Create a new project
+## Creating a New Project
 
-Let’s write a Ballerina program that prints “Hello, World!”. Use the `bal new` command to create a new Ballerina project. 
+Let’s write a Ballerina program, which prints `Hello, World!`. Use the `bal new` command to create a new Ballerina project. 
 
 ```bash
 $ bal new greeter
 ```
 
-This command creates a new directory called `greeter`. Let’s move into this directory to see what’s in there. 
+This command creates a new directory called `greeter` with the content below.
 
 ```bash
 greeter/
 ├── Ballerina.toml
 └── main.bal
 ```
 
-- `Ballerina.toml` contains metadata that describes your project. Also, `bal` tool uses `Ballerina.toml` to identify the root of a project.
+- `Ballerina.toml` contains metadata that describes your project. Also, the `bal` tool uses `Ballerina.toml` to identify the root of a project.
 - `main.bal` is a source file and it should contain Ballerina code that prints “Hello, World!” to the console. You can add any number of source files into the `greeter` directory.
 
-## Say “Hello, World!”
+## Saying `Hello, World!`
 
-You can open the project directory in your text editor. If you are using VSCode, run `code .` inside the `greeter` directory. Then, open `main.bal` to see the generated source.
+1. Open the project directory in your text editor. 
```

Comment:
Can we please keep the previous paragraph? I prefer that instead of these numbered points

Reply:
Fixed.